modelmapper/mapper.py

```diff
@@ -158,11 +161,22 @@ def hastype(self):
     SqlalchemyFieldType.Integer,
     SqlalchemyFieldType.BigInteger
 }
-NUMERIC_REMOVE = (',', '$', '%')
+
+NUMERIC_REMOVE = (',', '$', '%', '-')
+
+
+def _remove_extra_chars_from_number(item, absolute=False):
+    if item.startswith('(') and item.endswith(')'):
+        item = item.strip('()')
+        item = f'-{item}'
+    for i in NUMERIC_REMOVE:
+        if not(i == '-' and not absolute):
+            item = item.replace(i, '')
+    return item
 
```

Comment:
If we expect the data to be messy, why don't we just remove all combinations of delimiters instead of only checking for '()'